# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: uriloader/exthandler/win/nsMIMEInfoWin.h
# Commit: 1e4f8d9d96ce
# Full Hash: 1e4f8d9d96ce1ccb79c154416b119907f9479294
# Author: Mark Striemer <mstriemer@mozilla.com>
# Date: 2020-06-04 09:29:07
# Regressor Bug: 1632277
# File Overlap Count: 1
# Description:
#   Bug 1632277 - Part 1: Launch PDFs in app mode when default r=jaws,tkikuchi
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D73774
# ==============================================================================

diff -r 04d728d14947 -r 1e4f8d9d96ce uriloader/exthandler/win/nsMIMEInfoWin.h
--- a/uriloader/exthandler/win/nsMIMEInfoWin.h	Wed Jun 03 23:41:48 2020 +0000
+++ b/uriloader/exthandler/win/nsMIMEInfoWin.h	Wed Jun 03 18:21:32 2020 +0000
@@ -65,7 +65,8 @@
                    const nsAString& appFilesystemCommand);
 
   // Helper routine to call mozilla::ShellExecuteByExplorer
-  nsresult ShellExecuteWithIFile(nsIFile* aExecutable, const nsString& aArgs);
+  nsresult ShellExecuteWithIFile(nsIFile* aExecutable, int aArgc,
+                                 const wchar_t** aArgv);
 };
 
 #endif
